pkg/operator/resources/cert/factory.go

```diff
@@ -211,12 +211,17 @@ func createCertificateDefinitions() []CertificateDefinition {
 	}
 }
 
-func createSecret(name string) *corev1.Secret {
+func createTLSSecret(name string) *corev1.Secret {
 	return &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:   name,
 			Labels: utils.ResourceBuilder.WithCommonLabels(nil),
 		},
+		Type: corev1.SecretTypeTLS,
+		Data: map[string][]byte{
+			"tls.key": []byte(""),
+			"tls.crt": []byte(""),
```

Comment:
@mhenriks this feels so wrong but but not sure if we have alternatives as we must be the ones creating them?

Reply:
This doesn't offend me very much